# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../crm114/dont_profile.patch
# Copyright (C) 2008 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

the -pg option makes `crm` whine when it can't write a gmon.out file

--- ./Makefile.orig	2008-08-14 06:13:22.000000000 -0400
+++ ./Makefile	2008-08-14 06:13:48.000000000 -0400
@@ -81,7 +81,7 @@
 #    BSD users - do NOT use -static-libgcc!!!
 #LDFLAGS += -static -static-libgcc 
 #    use this one if you want program profiling.
-LDFLAGS += -static -static-libgcc -pg
+#LDFLAGS += -static -static-libgcc -pg
 #
 #   Any standard install flags?  We nominally use protection 755
 INSTALLFLAGS += -m 755
